fvtest/porttest/main.cpp

```diff
@@ -45,14 +46,18 @@ omr_main_entry(int argc, char **argv, char **envp)
 			strcpy(testName, &argv[i][7]);
 		} else if (0 == strcmp(argv[i], "-earlyExit")) {
 			earlyExit = true;
+		} else if (0 == strcmp(argv[i], "-verbose")) {
+			isVerbose = true;
 		}
 	}
 
 	if (!isChild) {
 		::testing::InitGoogleTest(&argc, argv);
 	}
 
-	OMREventListener::setDefaultTestListener();
+	if (!isVerbose) {
+		OMREventListener::setDefaultTestListener();
```

Comment:
I think we want to set the listener either way, but we can pass in some optional booleans to print more stuff:

```c++
OMREventListener::setDefaultTestListener(true, isVerbose, isVerbose, true);
```

function defined here: herehttps://github.com/eclipse/omr/blob/6eea70732bef0f9816ba902698cdf356b7502f32/fvtest/omrGtestGlue/omrTest.h#L124

I wonder if it'd be ok to just change the default values and enabled verbose printing for all test executables.